app/controllers/business_unit_kinds_controller.rb

```diff
@@ -0,0 +1,56 @@
+class BusinessUnitKindsController < ApplicationController
+  respond_to :html, :json
+
+  before_action :auth, :check_privileges
+  before_action :set_business_unit_kind, only: [:show, :edit, :update, :destroy]
+  before_action :set_title, except: [:destroy]
+
+  # GET /business_unit_kinds
+  def index
+    @business_unit_kinds = BusinessUnitKind.list.page(params[:page])
```

Comment:
```suggestion
    @business_unit_kinds = BusinessUnitKind.list.page params[:page]
```
Conviene también que esté con un orden.